src/main/kotlin/no/nav/familie/klage/behandling/BehandlingService.kt

```diff
@@ -54,6 +56,10 @@ class BehandlingService(
         ).id
     }
 
+    fun hentBehandlinger(eksternFagsakId: String, fagsystem: Fagsystem): List<Klagebehandlingsesultat> {
```

Comment:
Burde kanskje hete finnKlagebehandlingsresultat for å følge patternet med hvis det returneres en liste så pleier vi å prefixe med `finn..`, mens hvis det returneres kun en så prefixes det med `hent..`